packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -150,6 +165,37 @@ const stateReducer: Reducer<DesignerState, DesignerAction> = (draftState, action
       treeChanged = true;
       break;
 
+    case 'PROP_SELECT_CANCEL':
+      draftState.droppingElement = null;
```

Comment:
If `CANCEL` happens during move, the component being moved is lost. Undo works but wouldn't be better if `CANCEL` would rollback the move?